src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -249,7 +262,9 @@ export default function SpeedUpAndCancelSheet() {
         provider: currentProvider,
         transaction: fasterTxPayload,
       });
-      onSendTransactionCallback?.(hash);
+      if (tx?.ensRegistrationName) {
+        saveCommitTransactionHash(hash);
```

Comment:
here we check if the tx is a commit tx since we're adding this param from `dataAddNewTransaction` for the commit tx